### DISCUSSION TITLE
installing @percy/cli but cannot use/start percy exec

### DISCUSSION THREAD

**Discussion by jcharnley** [percy/cli#734] [Support] [answered]
as title details I cannot seem to run percy as the command is not found

added loads via npm & npx but nothing is working

"@percy/cli": "^1.0.0-beta.73",
"@percy/selenium-webdriver": "^1.0.0",
"@percy/cli-exec": "^1.0.0-beta.73",
"@percy/core": "^1.0.0-beta.73",

**Comment by Robdel12** [comment, marked as answer]
Hey @jcharnley! Can you provide more info about your steps for setting up, what's going wrong (errors, etc), your environment, and logs?

**Reply by jcharnley** [reply]
sure no problem. it's a JavaScript project, running selenium-webdriver. tests run fine but trying to use percy for screenshots and visual regression testing

I've created the project on percy dashboard, exported the token and just trying to run the file

packages i've installed are    
 "@percy/cli-exec": "^1.0.0-beta.73",
  "@percy/core": "^1.0.0-beta.73", 
    "@percy/cli": "^1.0.0-beta.73",
    "@percy/selenium-webdriver": "^1.0.0",


current version of selenium-webdriver i

    "selenium-webdriver": "^4.1.0",

As percy/cli docs mention starting the file using a start script I get

[percy] Percy is not running, disabling snapshots

and below says to start the percy server via exec (percy exec), but whatever I try the command is not found. 

Any Ideas?

**Reply by Robdel12** [reply]
Gotcha, thanks for that info! Did you `npm install` `@percy/cli-exec` and `@percy/core` manually? If so, you can remove those. You only need to install `@percy/selenium-webdriver` (the test framework SDK) and `@percy/cli` (what handles the SDK heavy lifting). 

With that said this log means the `PERCY_TOKEN` isn't set (rather than a command not found error):

```sh 
[percy] Percy is not running, disabling snapshots
```

You'll need to grab the `PERCY_TOKEN` from your Percy projects settings and then set that env var in the terminal you're running the CLI commands.

**Reply by jcharnley** [reply]
yeah I installed manually, ok that clears it up. ill follow your instructions thanks :)

**Reply by jcharnley** [reply]
awesome all done, you can close it. even tho I set it manually it wasn't picking it up. 

Maybe a good idea to add to docs as its easy to do via the package.json

    "percy-start": "PERCY_TOKEN=[token from .env] percy exec:start",
